CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 
 ##### Breaking
 
-* None.
+* Change `Text` enum case names to match Swift 3 API guidelines.
```

Comment:
This doesn't match the format specified by [CONTRIBUTING.md](https://github.com/jpsim/SourceKitten/blob/master/CONTRIBUTING.md). I'll take care of it this time.